src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4004,7 +4004,7 @@ bool os::can_commit_large_page_memory() {
 }
 
 bool os::can_execute_large_page_memory() {
-  return UseTransparentHugePages;
+  return UseLargePages;
```

Comment:
Could you please exclude this change from the PR? The function will be removed by my PR.

In `test/hotspot/jtreg/ProblemList.txt` could you please change
```
compiler/codecache/CheckLargePages.java 8317831 linux-x64
```

to
```
compiler/codecache/CheckLargePages.java 8319795 linux-x64
```